pkg/loadbalancer/experimental/service_annotations.go

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package experimental
+
+import (
+	"strings"
+
+	"github.com/cilium/cilium/pkg/annotation"
+	"github.com/cilium/cilium/pkg/loadbalancer"
+)
+
+func (svc *Service) GetLBAlgorithmAnnotation() loadbalancer.SVCLoadBalancingAlgorithm {
+	return loadbalancer.ToSVCLoadBalancingAlgorithm(svc.Annotations[annotation.ServiceLoadBalancingAlgorithm])
+}
+
+func (svc *Service) GetAnnotations() map[string]string {
+	return svc.Annotations
+}
+
+func (svc *Service) GetIncludeExternalAnnotation() bool {
+	if value, ok := annotation.Get(svc, annotation.GlobalService, annotation.GlobalServiceAlias); ok {
+		return strings.ToLower(value) == "true"
+	}
+	return false
+}
```

Comment:
Nit. It would probably make sense to eventually merge these functions with the already existing logic to parse these annotations (I assume calling directly into them would currently result in a circular import), just to prevent possible divergences in the future.

Reply:
Moved to pkg/annotation. This seems much nicer.